samples/multiplatform/androidApp/app/build.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
```

Comment:
since we don't have android app itself (only empty stub) is makes no sense to add extensions here

Reply:
It may be useful if a user wants to write an application on basis of this sample.

Reply:
AS will add this plugin automatically on creating new activity, but this is not a big deal anyway

Reply:
Ok, removed this.